src/app.jsx

```diff
@@ -34,19 +33,41 @@ export const Application = () => {
 
     const onFilterChange = (_, value) => setCurrentFilter(value);
 
-    // TODO: Start navigator on user home directory
     useEffect(() => {
         cockpit.user().then(user => {
             setCurrentUser(user.name || "");
-        })
-                .then(() => {
-                    const currentPath = path.slice(0, pathIndex).join("/");
-                    cockpit.spawn(["ls", "-p", `/${currentPath}`], { superuser: true }).then((res) => {
-                        res = res.split("\n");
-                        setFiles(res.slice(0, res.length - 1));
-                    });
-                });
-    }, [currentUser, path, pathIndex]);
+            setPath(["home", currentUser]);
```

Comment:
Martin suggested for this to use cockpit.user().then(user => user.home)
Can you please adjust it?